device/tt_xy_pair.h

```diff
@@ -12,3 +12,15 @@
 
 using tt_xy_pair = tt::umd::xy_pair;
 using tt_cxy_pair = tt::umd::cxy_pair;
+
+using tt_physical_coords = tt::umd::xy_pair;
+using tt_chip_physical_coords = tt::umd::cxy_pair;
+
+using tt_logical_coords = tt::umd::xy_pair;
+using tt_chip_logical_coords = tt::umd::cxy_pair;
+
+using tt_virtual_coords = tt::umd::xy_pair;
+using tt_chip_virtual_coords = tt::umd::cxy_pair;
+
+using tt_translated_coords = tt::umd::xy_pair;
+using tt_chip_translated_coords = tt::umd::cxy_pair;
```

Comment:
Can you add documentation that explains what these mean?

Reply:
So I have copied over docs from open-umd. I will go through it to refactor it a bit maybe, but in general all coordinate systems are described https://github.com/tenstorrent/tt-umd/blob/pjanevski/soc_desc_harvesting/docs/coordinate_systems.md